base/exports.jl

```diff
@@ -23,6 +23,7 @@ export
     Docs,
     Markdown,
     Threads,
+    Iterators,
```

Comment:
In that case yes, and you get the usual `WARNING: imported binding for Iterators overwritten in module Main`.